core/vesting/vesting_snapshot.go

```diff
@@ -64,21 +65,34 @@ func (e *SnapshotEngine) GetState(k string) ([]byte, []types.StateProvider, erro
 	return state, nil, err
 }
 
-func (e *SnapshotEngine) LoadState(_ context.Context, p *types.Payload) ([]types.StateProvider, error) {
+func (e *SnapshotEngine) LoadState(ctx context.Context, p *types.Payload) ([]types.StateProvider, error) {
 	if e.Namespace() != p.Data.Namespace() {
 		return nil, types.ErrInvalidSnapshotNamespace
 	}
 
 	switch data := p.Data.(type) {
 	case *types.PayloadVesting:
-		e.loadStateFromSnapshot(data.Vesting)
+		e.loadStateFromSnapshot(ctx, data.Vesting)
 		return nil, nil
 	default:
 		return nil, types.ErrUnknownSnapshotType
 	}
 }
 
-func (e *SnapshotEngine) loadStateFromSnapshot(state *snapshotpb.Vesting) {
+func (e *SnapshotEngine) recoverVesting736(ctx context.Context) {
```

Comment:
What does the 736 bit mean?

Reply:
Ah it's code for a specific version of the chain, would it be good to add a comment above this to explain that?

Reply:
I'd hazard a guess that's it's the v0.73.6 tag.